src/components/NavBar/NavigationBar.tsx

```diff
@@ -11,7 +11,7 @@ function NavigationBar(): JSX.Element {
   // separate because mobile should always be text, even
   // when nav bar is hidden
   const [isMobile, setIsMobile] = useState(
-    window.matchMedia('(max-width: 800px)')?.matches ?? false
+    window.matchMedia('(max-width: 47.9em)')?.matches ?? false
```

Comment:
Note: I use 47.9 em throughout because when using 48 em (which is the boundary for Chakra), sometimes the screen would render with the UI in desktop-mode but with the collapsible boxes still attached. Changing these numbers to 47.9 seems to have avoided that issue.